README.md

```diff
@@ -100,8 +100,8 @@ exports[`it works: b 1`] = `
 
 ## withStatechart(statechart[, options])(Component)
 
-The `withStatechart` higher-order component takes a statechart definition (see [xstate](https://github.com/davidkpiano/xstate)), some optional [options](#options) and a component.
-It returns a new component with special [props](#props), [action methods](#action-methods) and [lifecycle methods](#lifecycle-methods).
+The `withStatechart` higher-order component takes a statechart definition (see [xstate](https://github.com/davidkpiano/xstate)), some optional [options](#options) and a component. You can also set the `initial` machine state and `initialData` via props, e.g. `<withStatemachine(Component) initial="b" initialData={{ counter: 0 }} />`.
```

Comment:
I'd move `initialData` from options to props (and remove it from options) - I'm worried that having it in both places might be confusing.
Also, would you mind renaming `initial` to `initialMachineState`?
Thank you very much!

Reply:
Yes! I wanted to ask you that as well … will do